### PR TITLE
Update Simulation README.md

### PR DESCRIPTION
**Pull request by puzzler7**
The previous README had a link to the docker documentation (which commands to run, which image to use, etc.) which apparently no longer exists. Aside from that, the readme has been completely updated for the new docker setup.

### CODE REVIEW

File: simulation/README.md
Comment (muthuArivoli) [inline simulation/README.md:16]: You can delete all of this, the docker compose file takes care of it. If you want to say something, you can say start up the landside container as usual, or as located in the main readme.

File: simulation/README.md
Comment (muthuArivoli) [inline simulation/README.md:45]: It should be root@localhost. That will always work for landside.

File: simulation/README.md
Comment (muthuArivoli) [inline simulation/README.md:46]: Tell them to use the build script, as stated in the main readme. Honestly, I think you can delete these two steps (1 and 2), since it is something that you do every single time to want to use the landside container. You could also just point to the main readme and say follow that. @rabinowitz what are your thoughts?

File: simulation/README.md
Comment (muthuArivoli) [inline simulation/README.md:49]: Beneath this line, there is discussion about windows 10 pro and docker toolbox. Delete all of that.

**Review by muthuArivoli — CHANGES_REQUESTED** [review]

File: simulation/README.md
Comment (puzzler7) [reply simulation/README.md:39]: This is my bad, I looked and looked for docker documentation in the various repos but failed to check the main readme. I think step 2 should go, but it's worth keeping step 1 and saying that they should be in the landside container.

I'll also make further changes to the readme when I do the code cleanup, which will almost certainly not be done tonight, but hopefully tomorrow.

File: simulation/README.md
Comment (puzzler7) [reply simulation/README.md]: Fixed.

File: simulation/README.md
Comment (puzzler7) [reply simulation/README.md]: Fixed.

File: simulation/README.md
Comment (puzzler7) [reply simulation/README.md:42]: Deleted.

File: simulation/README.md
Comment (muthuArivoli) [reply simulation/README.md]: not quite, **root** at localhost, not duke

Comment (puzzler7) [reply simulation/README.md]: Fixed for real, now.

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:8]: Change to assuming the containers are running for [local testing](https://github.com/DukeRobotics/robosub-ros#local-testing) as specified in the main README. The current assumptions should no longer need to be stated as long as they've followed the instructions in the main readme.

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:12]: This link no longer works. It is now just generically https://coppeliarobotics.com/downloads for all OSes. I'd recommend having a sentence that says "Download the Edu version from [here](https://coppeliarobotics.com/downloads)." before the OS-specific instructions. Then it's up to you whether you think there's enough separate information to justify a 4th-level header for each OS, or if you should do a bullet-list to make it more compact and readable.

Comment (rabinowitz) [inline simulation/README.md:12]: Comments apply to following lines since I can't seem to comment on those.
Nit: Mac should be capitalized or written as "macOS" in step 1 of the Mac instructions.
Nit: CoppeliaSim should be capitalized in step 3.
Nit: Python should be capitalized in all uses in the Downloading Remaining Files section. There are two uses I see that need to be fixed.

Comment (rabinowitz) [inline simulation/README.md:12]: The heading below should read "Downloading on Ubuntu" since the downloads page only lists Ubuntu, not Linux in general.

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:40]: Not something that should hold up this PR, but is there a way we can fix our files or Docker images so you don't have to run `chmod` to run that executable? @muthuArivoli maybe you have an idea?

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:41]: nit: Python should be capitalized.

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:50]: nit: the line below this should have Windows capitalized both times it is mentioned.

File: simulation/README.md
Comment (rabinowitz) [inline simulation/README.md:47]: Rather than specify Python 3 like that, maybe just change the command to include Python 3. Everyone should be using Python 3 now anyway. I've reworded it here for some other clarity.
```suggestion
4. In `robosub-ros/simulation/personal` on your local machine, run `python3 run_coppelia_comm.py`.
```

**Review by rabinowitz — CHANGES_REQUESTED** [review]
A few suggestions that clarify instructions and fix dead links, and a few nitpicky things.

**Comment by rabinowitz** [comment]
I'm also going to commit a couple of teensy fixes to the main readme that are tangentially related to this PR and I was waiting for an excuse to do without their own PR.

File: simulation/README.md
Comment (muthuArivoli) [inline simulation/README.md:34]: The problem with this is that roscore should only be run in the onboard container. By doing the
 `source /opt/ros/melodic/setup_network.bash` you are saying that the roscore is running in the onboard container. I think you can do something like:
1. In the onboard container, make sure `roscore` is running.
2. Now, in the landside container, run ...

**Comment by rabinowitz** [comment]
LGTM! Make sure to "Squash and merge" once Muthu approves.

**Review by rabinowitz — APPROVED** [review]
LGTM! Make sure to "Squash and merge" once Muthu approves.

**Review by muthuArivoli — APPROVED** [review]
LGTM

**Pull request merged**
